frontend/src/features/rollout-announcement/components/ProgressIndicator.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box } from '@chakra-ui/react'
+import { motion } from 'framer-motion'
+
+const ActiveIndicator = (): JSX.Element => (
+  <Box
+    width="1.5rem"
+    height="0.5rem"
+    borderRadius="full"
+    backgroundColor="#5D6785"
+  />
+)
+
+interface InactiveIndicatorProps {
+  onClick?: () => void
+  isActiveIndicator: boolean
+}
+
+const CircleIndicator = (props: InactiveIndicatorProps): JSX.Element => {
+  const { onClick, isActiveIndicator } = props
+  return (
+    <Box
+      width="0.5rem"
+      height="0.5rem"
+      borderRadius="full"
+      backgroundColor="#BABECB"
+      marginRight={isActiveIndicator ? '1.5rem' : '0.5rem'}
+      onClick={onClick}
+    />
+  )
+}
+
+interface ProgressIndicatorProps {
+  numIndicators: number
+  currActiveIdx: number
+  onClick?: (indicatorIdx: number) => void
```

Comment:
why make this optional?

Reply:
Initially thought there could be a scenario where the progress indicators don't need to be clickable. But now I think it wouldn't be consistent design-wise. I've updated the PR so that `onClick` is no longer optional